### DISCUSSION TITLE
congratulations for 3k downloads

### DISCUSSION THREAD

**Discussion by ihatemyself0** [z44d/tgram#18] [General]
congratulations for 3k downloads and i hope everything will be fixed soon

![tenor](https://github.com/2ei/tgram/assets/173957402/68128376-2d99-4467-b3cb-32b444c9b319)

**Comment by z44d** [comment]
Thanks

**Reply by a7a-dar** [reply]
hello

**Reply by z44d** [reply]
wtf
